modules/apps/commerce/commerce-order-content-web/src/main/java/com/liferay/commerce/order/content/web/internal/health/status/CommerceGuestCheckoutAuthenticationCommerceHealthStatus.java

```diff
@@ -144,7 +144,7 @@ public void fixIssue(long companyId, long commerceChannelId)
 
 			List<JournalArticle> journalArticles =
 				_journalArticleLocalService.getArticlesByStructureId(
-					commerceChannel.getSiteGroupId(),
+					serviceContext.getScopeGroupId(),
```

Comment:
@andrea-ale-sbarra I think this should be in a separate commit, not part of the build lang.

Reply:
Yes, thank you @dsitu I did a mistake by adding that class in this commit, I've removed it.